components/dashboard/src/data/git-providers/suggested-repositories-query.ts

```diff
@@ -24,6 +24,9 @@ export const useSuggestedRepositories = ({ excludeConfigurations }: Props) => {
             const { repositories } = await scmClient.listSuggestedRepositories({
                 organizationId: org.id,
                 excludeConfigurations,
+                pagination: {
+                    pageSize: 100,
```

Comment:
We now query at most 100 projects (SCM repos not included) for the initial load in repo selectors. Pagination is not implemented, but also not necessary yet